Source/PLCrashNamespace.h

```diff
@@ -318,3 +318,6 @@
 #  define protobuf_c_version                                PLNS(protobuf_c_version)
 #  define protobuf_c_version_number                         PLNS(protobuf_c_version_number)
 #endif /* PLCR_PRIVATE */
+
+#pragma clang diagnostic push
```

Comment:
1. `push` must be paired with `pop`
2. It's ignoring, not a "fix"
3. This header is NOT supposed to be used for such things

Reply:
Yes, at the moment it is more of a draft. There is an idea to do this (ignoring) using a `.pch` file. In the case of a fix, not ignoring, there will be more than 17 files. Most likely twice as many files will be affected.